o1vm/src/pickles/main.rs

```diff
@@ -1,6 +1,6 @@
 use ark_ff::UniformRand;
 use clap::Parser;
-use kimchi::circuits::domains::EvaluationDomains;
+use kimchi::{circuits::domains::EvaluationDomains, precomputed_srs::TestSRS};
```

Comment:
Out of the scope of this PR: it would be nice to move this structure `TestSRS` in `poly_commitment` in `ipa.rs`, next to `SRS`. We should also have a unified structure for `TestSRS` and `SRS`. It is mostly the encoding the difference. I guess we can have a parameter or something to serde.